Authentication/Extensions.swift

```diff
@@ -16,11 +16,11 @@ internal extension String {
          not have the key in the framework one.
      */
     internal var frameworkLocalized: String {
-        let localized: (from: NSBundle) -> String = {
+        let localized: (_ from: Bundle) -> String = {
```

Comment:
Yeah, I also think this behavior is strange, but I checked Core's Swift 3 migration PR to be sure and the only thing I changed from that function was the order of the parameters, so if this worked properly in Swift 2, then maybe they changed it in Swift 3?